docs/guides/setup-dbt.mdx

```diff
@@ -161,7 +161,7 @@ For the rest of this tutorial, we’ll use the project name `demo_project`.
        , b.*
    FROM {{ ref('my_first_dbt_model') }} AS a
 
-   LEFT JOIN {{ source('mage_demo', 'dbt_demo_pipeline_load_data') }} AS b
+   LEFT JOIN {{ source('mage_dbt_demo_pipeline', 'mage_dbt_demo_pipeline_load_data') }} AS b
```

Comment:
<img width="951" alt="image" src="https://github.com/mage-ai/mage-ai/assets/80284865/14373710-d793-417f-b34a-f255943972b5">

I got this error when using `mage_dbt_demo_pipeline_load_data` as the table name here.
I have to change it to `{{ source('mage_dbt_demo_pipeline', 'load_data') }}`